app/src/main/java/ca/bc/gov/bchealth/compose/Styles.kt

```diff
@@ -54,15 +54,15 @@ val MyHealthTypography = Typography(
 
     button = TextStyle(
         fontFamily = fonts,
-        fontWeight = FontWeight.Bold,
+        fontStyle = FontStyle.Normal,
         fontSize = 16.sp,
-        color = primaryBlue
+        color = white
```

Comment:
This textStyle is already being used in the project, did you check if they are still working properly with this change?

Reply:
I checked this styles are only used in the two places and that is also under my new developed screen

Reply:
Just checked here and CommentInputUI and FeedbackUI are using the button style on a white background, could you update them please?